.github/workflows/job-postcommit-python.yml

```diff
@@ -0,0 +1,77 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# Test for Python Postcommit
+
+
+name: Python Postcommit
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 */6 * * *'
+  push:
+    branches: ['master', 'release-*']
+    tags: ['v*']
+    paths: [ 'sdks/python/apache_beam/testing/test_stream_it_test.py',
+             'sdks/python/apache_beam/transforms/external_it_test.py']
+permissions: read-all
+
+env:
+  tests: "'apache_beam/testing/test_stream_it_test.py \
+  apache_beam/transforms/external_it_test.py'"
+
+jobs:
+  set-properties:
+    runs-on: [self-hosted, ubuntu-20.04]
+    outputs:
+      properties: ${{ steps.test-properties.outputs.properties }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          persist-credentials: false
+      - id: test-properties
+        uses: ./.github/actions/setup-default-test-properties
+
+  py_postcommit:
+    needs: set-properties
+    name: Python Postcommit Tests
```

Comment:
Does this contain all Python Postcommit tests? If yes, should we be deleting the Jenkins file? If no, could we give this a more descriptive name?

Reply:
Hi @damccorm 
No, it does not contain all the python postcommit, since we are splitting the python postcommit jobs as indicates the [following file](https://docs.google.com/spreadsheets/d/1PbEPSWqQfjLGLUBMUctjKF2tuu2AYeAcSoiP_OL71B4/edit#gid=639595669&range=41:41), that's why this only contain this jobs, we can rename the workflow so it does not confuse the people...

Reply:
Maybe we can just rename to Python PostCommit Cloud and Cython Tests?